src/api/services/kcUsersService.ts

```diff
@@ -40,3 +43,35 @@ export const sendInviteEmail = async (
     redirectUri: SSP_WEB_BASE_URL,
   });
 };
+
+export const deleteUser = async (kcAdminClient: KeycloakAdminClient, user: UserRepresentation) => {
+  await kcAdminClient.users.del({
+    id: user.id!,
+  });
+};
+
+export interface KcUserRequest extends UserRequest {
+  kcUser?: UserRepresentation;
+  kcAdminClient?: KeycloakAdminClient;
```

Comment:
I don't think providing something like a keycloak client is appropriate using an enricher like this. It's nothing to do with the request itself.

Reply:
I see the reasoning - you don't want to auth twice. But putting a rich object like a client on the request seems like a bad idea - the request object is for data related to the request.

Reply:
Yep, I agree, it doesn't make sense to include that object in the request. Removed